bigchaindb/common/transaction.py

```diff
@@ -1248,19 +1251,15 @@ def __str__(self):
         tx = Transaction._remove_signatures(self.to_dict())
         return Transaction._to_str(tx)
 
-    @classmethod
-    # TODO: Make this method more pretty
-    def from_dict(cls, tx_body):
-        """Transforms a Python dictionary to a Transaction object.
+    @staticmethod
+    def validate_structure(tx_body_orig):
+        """Validate the structure of a transaction body.
 
             Args:
                 tx_body (dict): The Transaction to be transformed.
-
-            Returns:
-                :class:`~bigchaindb.common.transaction.Transaction`
```

Comment:
That should go for the whole file then, indeed the whole codebase. I'd put it in a separate issue.